transforms/universal/hap/python/README.md

```diff
@@ -0,0 +1,54 @@
+# HAP Annotation
+Please see the set of [transform project conventions](https://github.com/ian-cho/data-prep-kit/blob/dev/transforms/README.md) for details on general project conventions, transform configuration, testing and IDE set up.
+
+## Prerequisite
+This repo needs NLTK and please refer to `requirements.txt`.
+
+## Summary
+The hap transform maps a non-empty input table to an output table with an added `hap_score` column. Each row in the table represents a document, and the hap transform performs the following three steps to calculate the hap score for each document:
+
+* Sentence spliting: we use NLTK to split the document into sentence pieces.
+* Hap annotation: each sentence is assigned a hap score between 0 and 1, where 1 represents hap and 0 represents non-hap.
+* Aggregation: the document hap score is determined by selecting the maximum hap score among its sentences.
+
+
+## Configuration and command line Options
+The set of dictionary keys holding [HAPTransformConfiguration](src/hap_transform.py) 
+configuration for values are as follows:
+
+* --model_name_or_path - specifies HAP model which should be compatable with HuggingFace's `AutoModelForSequenceClassification` 
+* --batch_size - modify it based on the infrastructure capacity.
+* --max_length - the maximum length for the tokenizer.
+
+
+
+## input format
+The input is in .parquet format and contains the following columns:
+
+| doc_id  |   doc_text | 
```

Comment:
The code does not seem to require doc_id.
And `contents` is the defacto standard name for the column containing the document.  Can you change that?

Reply:
@daw3rd Thanks for suggestions. In the sample parquet file in the `test-data/input`, there are such names `doc_id` and `doc_text`. Does it look very intuitive to users when they see such readme? If we want to change it into `contents`, we loss the information about whether the input is a doc or just a sentence.

Reply:
@Ian-cho you can continue to use doc_text in the sample parquet file since you are setting the name of the column in all of your tests. However, when folks use your code **without setting the content_name_column** I'd like that to default to "contents". Line 135 of hap_transform.py